clang/lib/Driver/Driver.cpp

```diff
@@ -4686,13 +4686,28 @@ class OffloadingActionBuilder final {
           else
             continue;
         } else if (A->getOption().matches(options::OPT_Xsycl_backend)) {
```

Comment:
Wouldn't this also be an issue with `-Xsycl-target-link` and `-Xsycl-target-frontend`?

Reply:
I initially thought it would, but then this code seems to deal exclusively with `--offload-info` arguments (see #4239). At the time checks are made for `-frontend` and `-linker`, the toolchain count expectations are normalized.